Tella/Scenes/CommonConnectionReport/ViewModels/ReportCardViewModel.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  ReportCardViewModel.swift
+//  Tella
+//
+//  Created by Dhekra Rouatbi on 1/7/2024.
+//  Copyright © 2024 HORIZONTAL. All rights reserved.
+//
+
+import Foundation
+
+
+class ReportCardViewModel: CommonCardViewModel {
+    
+    var status : ReportStatus = ReportStatus.unknown
+    
+    init(report : BaseReport,
+         serverName : String?,
+         deleteReport: @escaping (() -> Void)) {
+        
+        let title = report.title ?? ""
+        let serverName = serverName
+        let iconImageName : String? = report.status.iconImageName
+        let listActionSheetItem = report.status.listActionSheetItem
+        let deleteReportStrings = report.status.deleteReportStrings
+        
+        super.init(id: Int(UUID().uuidString) ?? 0,
```

Comment:
Why are we initializing the id like this? Shouldn't we initialize it with the id of the report?

Reply:
yeah, you're right it's a mistake